src/kw_db.sh

```diff
@@ -152,8 +152,8 @@ function format_values_db()
 
   for val in "$@"; do
     [[ "$count" -eq 0 ]] && values+='('
-    # check if not a sqlite function
-    if [[ ! "$val" =~ ^[[:alnum:]_]+\(.*\)$ ]]; then
+    # check if not a sqlite function nor NULL
```

Comment:
Could you expand this function documentation to add one example about this 'correct format'?